src/hosts/BitbucketAdapter.js

```diff
@@ -129,21 +100,43 @@ const BitbucketAdapter = (config: Config) => {
       return true;
     },
 
-    async getPullRequest(pullRequestId: string) {
+    async getPullRequest(pullRequestId: string): Promise<PullRequest> {
       const endpoint = `${apiBaseUrl}/pullrequests/${pullRequestId}`;
       const resp = await axios.get(endpoint, axiosGetConfig);
-      return resp.data;
+      const data = resp.data;
+      const approvals = data.participants
+        .filter(participant => participant.approved)
+        .map(participant => participant.user.username);
+
+      return {
+        pullRequestId: pullRequestId,
+        title: data.title,
+        description: data.description,
+        createdOn: new Date(data.created_on),
+        author: data.author.username,
+        state: data.state,
+        approvals: approvals,
+        openTasks: data.task_count
```

Comment:
What about GitHub, you want to just put 0 there in the case of GH?

Reply:
Yea, that would be nice